boards/arm/samv7/common/Kconfig

```diff
@@ -0,0 +1,96 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfiglanguage.txt in the NuttX tools repository.
+#
+
+config BOARD_SAMV7_COMMON
+	bool "Board common logic"
+	default y
+	help
+		 Board common logic located in each board/common folder.
+
+if BOARD_SAMV7_COMMON
+
+config SAMV7_PROGMEM_OTA_PARTITION
+	bool
+	default n
+	select ARCH_RAMFUNCS
+	select BCH
+	select MTD
+	select MTD_BYTE_WRITE
+	select MTD_CONFIG
+	select MTD_PARTITION
+	select MTD_PROGMEM
+	select MTD_PROGMEM_ERASESTATE
+	select SAMV7_PROGMEM
+	select SAMV7_PROGMEM_ERASESTATE
+
+config SAMV7_MCUBOOT_HEADER_SIZE
+	hex
+	default 0x200
+	depends on SAMV7_FORMAT_MCUBOOT
+
+menuconfig SAMV7_FORMAT_MCUBOOT
+	bool "MCUboot bootable format"
+	default n
+	select SAMV7_PROGMEM_OTA_PARTITION
+	help
+		 The MCUboot support of loading the firmware images.
+
+if SAMV7_FORMAT_MCUBOOT
+
+config SAMV7_MCUBOOT_BOOTLOADER
+	bool "MCUboot bootloader application"
+	default n
+	select BOARDCTL
+	select BOARDCTL_BOOT_IMAGE
+	help
+		Build the MCUboot first stage bootloader application. This
+		application is the first application that will run after reset.
+		To build normal applications that MCUboot should test and run,
+		left this option untouched.
+
+comment "MCUboot Application Image OTA Update support"
+
+config SAMV7_OTA_PRIMARY_SLOT_OFFSET
+	hex "MCUboot application image primary slot offset"
+	default "0x20000"
+
+config SAMV7_OTA_PRIMARY_SLOT_DEVPATH
+	string "Application image primary slot device path"
+	default "/dev/ota0"
+
+config SAMV7_OTA_SECONDARY_SLOT_OFFSET
+	hex "MCUboot application image secondary slot offset"
+	default "0x48000"  if SAMV7_MEM_FLASH_512
+	default "0x80000"  if SAMV7_MEM_FLASH_1024
+	default "0x100000" if SAMV7_MEM_FLASH_2048
+
+config SAMV7_OTA_SECONDARY_SLOT_DEVPATH
+	string "Application image secondary slot device path"
+	default "/dev/ota1"
+
+config SAMV7_OTA_SLOT_SIZE
+	hex "MCUboot application image slot size (in bytes)"
+	default "0x28000"  if SAMV7_MEM_FLASH_512
+	default "0x60000"  if SAMV7_MEM_FLASH_1024
+	default "0xe0000"  if SAMV7_MEM_FLASH_2048
+
+config SAMV7_OTA_SCRATCH_OFFSET
+	hex "MCUboot scratch partition offset"
+	default "0x70000"  if SAMV7_MEM_FLASH_512
+	default "0xe0000"  if SAMV7_MEM_FLASH_1024
+	default "0x1e0000" if SAMV7_MEM_FLASH_2048
+
+config SAMV7_OTA_SCRATCH_DEVPATH
+	string "Scratch partition device path"
+	default "/dev/otascratch"
+
+config SAMV7_OTA_SCRATCH_SIZE
+	hex "MCUboot scratch partition size (in bytes)"
+	default "0x10000"  if SAMV7_MEM_FLASH_512
+	default "0x20000"
```

Comment:
```suggestion
	default 0x20000
```

Reply:
Is dependency on `SAMV7_MEM_FLASH_1024` and `SAMV7_MEM_FLASH_2048` missing or `0x20000` is intentional value for both?